auth/src/main.rs

```diff
@@ -30,19 +31,20 @@ async fn main() {
         .init();
 
     let db_path = args.state.join("authentication.sqlite");
-    let db_uri = db_path.to_str().unwrap();
 
-    println!("db_uri: {db_uri}");
-    // if !db_path.exists() {
-    //     Sqlite::create_database(db_uri).await.unwrap();
-    // }
+    info!(
+        "state db: {}",
+        std::fs::canonicalize(&args.state)
+            .unwrap()
+            .to_string_lossy()
+    );
 
-    // info!(
-    //     "state db: {}",
-    //     std::fs::canonicalize(&args.state)
-    //         .unwrap()
-    //         .to_string_lossy()
-    // );
+    let db_uri = db_path.to_str().unwrap();
 
-    start(args).await;
+    // If the start command is called, start the auth server with given DB path and address.
+    // If the init command is called, do the same but insert an admin user as well.
+    match args.command {
+        Commands::Start => start(db_uri, args.address, None).await,
+        Commands::Init(init_args) => start(db_uri, args.address, Some(init_args)).await,
```

Comment:
Maybe init should only edit the DB and exit. And not start up the whole API?

Reply:
Yup, with your router suggestion for tests this was doable. Thanks!